frontend/public/style/_layout.scss

```diff
@@ -2,138 +2,71 @@ $sidebar-width: 220px;
 $masthead-height-mobile: 40px;
 $masthead-height-desktop: 60px;
 
-html {
-  height: 100%;
-}
-
-body {
-  display: flex;
-  flex-direction: column;
-  min-height: 100vh;
-}
-
-#app {
-  display: flex;
-  height: 100%;
-  flex-direction: column;
-  flex: 1 0 auto;
-  overflow: hidden;
-}
-
-#reflex {
-  display: flex;
-  flex: 1 1 auto;
+html,
+body,
+#app,
+#content {
+  height: 100%; // so .co-p-has-sidebar, .yaml-editor are full height
 }
 
 #content {
-  overflow-y: auto;
-  -webkit-overflow-scrolling: touch; // Scroll with momentum
-  flex: 1 1 auto;
-  @media (max-width: $grid-float-breakpoint) {
-    padding-top: $masthead-height-mobile;
-  }
+  padding-top: $masthead-height-mobile;
```

Comment:
I suspect the height: 100% combined with padding-top might be causing some issues with react-virtualized. On the overview page, the top event is hidden too early:
![nshroqf](https://user-images.githubusercontent.com/200121/40147060-e47d523c-591c-11e8-9904-573bf0df98e8.png)

I see this in both chrome on linux and safari on mac.

Reply:
Good eye, we'll look at it.

I've seen a similar problem before, but I think it only happened occasionally at smaller screen widths.

Reply:
> I suspect the height: 100% combined with padding-top might be causing some issues with react-virtualized. 

[This was happening before](https://jira.coreos.com/browse/CONSOLE-471) this PR merged, but at a different viewport size.

Reply:
And it looks like it was also happening before this PR merged at roughly the same viewport size as seen in @ggreer's screenshot. 

![screen shot 2018-05-17 at 9 14 15 am](https://user-images.githubusercontent.com/895728/40179640-d77c414c-59b2-11e8-8f40-ea5498b65cfc.PNG)